lib/srv/db/elasticsearch/engine.go

```diff
@@ -156,11 +163,19 @@ func copyRequest(ctx context.Context, req *http.Request) (*http.Request, error)
 // process reads request from connected elasticsearch client, processes the requests/responses and send data back
 // to the client.
 func (e *Engine) process(ctx context.Context, sessionCtx *common.Session, req *http.Request, client *http.Client) error {
-	reqCopy, err := copyRequest(ctx, req)
+	body, err := io.ReadAll(io.LimitReader(req.Body, teleport.MaxHTTPRequestSize))
```

Comment:
This does not return an error when the limit is reached. I think the "limited" body should only be used for auditing and we should not send the "limited" body to the server. Or we can error out here. What do you think?

Reply:
If we audit limited body but send to the server full one, there technically possibility of hiding something from the audit though?

Reply:
We send the truncated copy over to the server, see below.